.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - oraclejdk7
+  - oraclejdk8
 before_install:  
  - "echo $JAVA_OPTS"
  - "export JAVA_OPTS=-Xmx512m"
```

Comment:
@acoburn I'm a dope! I misread the hyphens at the start of those lines as minus signs indicating deletions!